### PR TITLE
Add French sub-territories support

### PR DESCRIPTION
**Pull request by kaldebert**
Replaced PR #65.

### CODE REVIEW

File: src/main/java/org/iban4j/bban/BbanStructure.java [hunk 1]
```diff
@@ -34,6 +34,13 @@ private BbanStructure(final BbanStructureEntry... entries) {
 
     private static final EnumMap<CountryCode, BbanStructure> structures;
 
+    /* French sub-territories may use their own country code (BL,RE,NC,...) or FR for their IBAN. Structure is the same, only the IBAN checksum differ. */
+    private final static BbanStructure FRENCH_STRUCTURE = new BbanStructure(
+        BbanStructureEntry.bankCode(5, 'n'),
+        BbanStructureEntry.branchCode(5, 'n'),
+        BbanStructureEntry.accountNumber(11, 'c'),
+        BbanStructureEntry.nationalCheckDigit(2, 'n'));
+
```
Comment (hajk1) [inline src/main/java/org/iban4j/bban/BbanStructure.java:43]: Why did you add new structure here? As for as I see this is the same structure as CountryCode.FR.
You could make use of it and did not break the 2 letter abbreviation standard.
Is there any reason behind this?

Comment (FrankHossfeld) [reply src/main/java/org/iban4j/bban/BbanStructure.java:43]: What kind of solution would you prefer/expect?

**Comment by kaldebert** [comment]
@hajk1 Hi ! Do you plan to release a new version with this fix?

**Comment by hajk1** [comment]
Yes, I will release new version in the coming week

**Comment by kaldebert** [comment]
Hi @hajk1 !
Do you have some news about the next release ?

**Comment by FrankHossfeld** [comment]
@kaldebert 
if you can not wait for a new release, you can try: https://github.com/NaluKit/iban4g. The current released version does already contain the bug fix. API is the same. At least it's based on the code of this repo. The only difference is, that there are some internal changes, so that the GWT & J2CL Transpiler can transpile the Java code. This changes should not have an impact when used in a pure Java environment.

**Comment by hajk1** [comment]
> Hi @hajk1 !
> Do you have some news about the next release ?

Version 3.2.3-RELEASE has been released and the maven repository will be updated during the next hours.
